terraform/variables.tf

```diff
@@ -39,6 +39,7 @@ variable "image_tag" {
   default     = "latest_dynamic"
 }
 
+# This var is used by cluster test in cluster.rs, please update in both places if this value changes
```

Comment:
I'm willing to bet 5 Libra USD that this will break cluster-test at least once :)